tox.ini

```diff
@@ -14,17 +14,6 @@ skip_install = true
 deps = check-manifest
 commands = check-manifest
 
-# Docs
```

Comment:
I would still like to keep this, as it let's me test that the docs are alright before making PRs